lib/src/options.dart

```diff
@@ -0,0 +1,17 @@
+class Options {
```

Comment:
This doesn't belong in a generic library as it is only applicable to HtmlGenerator.  So it should be defined in html_generator.dart (remember, cyclic imports are allowed in dart) since it is part of the interface to HtmlGenerator, and should be called something less generic, like HtmlGeneratorParameters.

However, I'd prefer we just not do this at all; while shortening the parameter list is a noble goal, we haven't actually achieved that in any meaningful sense.  We've just moved the construction of the argument list around a bit, and since this is the only place and the only object that uses this option field it is really just more readable to have an argument list.